WordPress/WordPressTest/RootViewCoordinatorTests.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import XCTest
+
+@testable import WordPress
+
+final class RootViewCoordinatorTests: XCTestCase {
+    private var sut: RootViewCoordinator!
+    private var windowManager: WindowManagerMock!
+
+    override func setUp() {
+        super.setUp()
+        windowManager = WindowManagerMock(window: .init())
+        sut = RootViewCoordinator(
+            featureFlagStore: RemoteFeatureFlagStoreMock(),
+            windowManager: windowManager,
+            wordPressAuthenticator: WordPressAuthenticatorMock.self
+        )
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        windowManager = nil
+        sut = nil
+    }
+
+    func testAppUIDeallocatedAfterLogout() {
```

Comment:
Nice test! It's good to be able to test that things are deallocated after logout.